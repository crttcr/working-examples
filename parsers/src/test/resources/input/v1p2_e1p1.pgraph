(v:PERSON 
	{ name = "Donald", 
	  color = "CHEETOH"
	}
)

(v)---[:LOVES {level="Intense"}]--->(v)
