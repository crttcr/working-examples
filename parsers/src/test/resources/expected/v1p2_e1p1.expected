VERTEX: v
	LABEL: PERSON
		PROP name = "Donald"
		PROP color = "CHEETOH"
EDGE: LOVES
		PROP level = "Intense"
		